kie-wb-parent/pom.xml

```diff
@@ -241,6 +241,31 @@
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>org.wildfly.swarm</groupId>
+          <artifactId>wildfly-swarm-plugin</artifactId>
+          <configuration>
+            <fractionDetectMode>never</fractionDetectMode>
+            <fractions>
+              <fraction>logging</fraction>
+              <fraction>undertow</fraction>
+              <fraction>cdi</fraction>
+              <fraction>ejb</fraction>
+              <fraction>jaxrs</fraction>
+              <fraction>ee</fraction>
+              <fraction>security</fraction>
+              <fraction>management</fraction>
+              <fraction>datasources</fraction>
+            </fractions>
```

Comment:
Do both WARs need exactly the same fractions? It seems that the runtime/monitoring WAR should not need all of these? Adding more fractions means the uberjar will be bigger. I would opt for keeping the number of fractions to a only those that are needed.

Reply:
@psiroky I do agree with your concern but at this stage, I believe both distributions have the same dependencies.